eduvpn/i18n.py

```diff
@@ -9,9 +13,82 @@
 country_mapping = None
 
 
+def init(lets_connect: bool, prefix: str):
+    """
+    Init locale and gettext, returns text domain
+    """
+    domain = 'LetConnect' if lets_connect else 'eduVPN'
+    directory = os.path.join(prefix, 'share/locale')
+
+    locale.setlocale(locale.LC_ALL, '')
+    locale.bindtextdomain(domain, directory)  # type: ignore
+    locale.textdomain(domain)  # type: ignore
+    install(domain, localedir=directory)
```

Comment:
I would prefer we don't install `_()` globally like this, but use explicit imports instead.

Reply:
Do you mean something like?
```
from gettext import as _
```
if so, I tried that in advance, but unfortunately that does not work. See
https://stackoverflow.com/questions/46394865/python-i18n-with-gettext-not-working

Reply:
Yes, that's what I mean. What do you mean doesn't work? `xgettext` should include it in the `.mo` file since it only looks at the function name, and the installed `_(...)` should be the same as `gettext.gettext(...)`.

Reply:
Was I mean is: gettext translations only work if the functions `_`, `gettext` and `ngettext` are installed in the builtins namespace.
If e.g. `from gettext import _` is used, simply no translation takes place.
This problem is also described [here](https://stackoverflow.com/questions/46394865/python-i18n-with-gettext-not-working) and I can reproduce this exactly with Python 3.8.5.

Reply:
You still need to configure the domain and localedir for gettext to be able to resolve the translation files. Replacing the call to `install(...)` with [`translation(...)`](https://docs.python.org/3/library/gettext.html#gettext.translation) should work.

Reply:
This is outdated code. For the current code see See [here](https://github.com/eduvpn/python-eduvpn-client/pull/341/files):
```
install(domain, names=('gettext', 'ngettext'), localedir=directory)
```
This installs `gettext` and `ngettext` in the  builtins namespace.

And this seems to be the "only one way, albeit the most convenient way, to make the _() [and `gettext` and `ngettext`] function available to your application".
https://docs.python.org/3/library/gettext.html#gettext.NullTranslations.install